src/main/java/io/soracom/krypton/SORACOMKryptonCLI.java

```diff
@@ -61,6 +62,7 @@
 		kryptonOptionHanderList.add(new GenerateAmazonCognitoOpenIdTokenOperationHandler());
 		kryptonOptionHanderList.add(new BootstrapAwsIotThingOperationHandler());
 		kryptonOptionHanderList.add(new BootstrapAzureIotDeviceOperationHandler());
+    kryptonOptionHanderList.add(new BootstrapArcOperationHandler());
```

Comment:
Could you add an indent and align the formatting?

Reply:
Oops.. Sorry @c9katayama it seems I lost track of this a while ago. I have added the proper indentation now.